docs_user/modules/proc_adopting-the-placement-service.adoc

```diff
@@ -41,13 +40,13 @@ spec:
 
 .Verification
 
-* Check that Placement endpoints are defined and pointing to the
-control plane FQDNs and that Placement API responds:
+* Check that the Placement service endpoints are defined and pointing to the
+control plane FQDNs, and that the Placement API responds:
 +
 ----
-alias openstack="oc exec -t openstackclient -- openstack"
+$ alias openstack="oc exec -t openstackclient -- openstack"
```

Comment:
as a genera comment adding `$` to the start of a line to indifcat a prompt signifcally reduced the ux of using the docs as you cannot simply copy paste the lines simply.

the rendering hack that tries to remove those automatically in some cases does not help with this. escipally not in the pdf version fo the docs or when they are sued directly form github.

so i would reconsider making this change and reevaluate the style rule that is driving it.

Reply:
I agree with Sean

Reply:
I will bring this up at my next team meeting. This change would affect all documentation, and I shouldn't make this decision on my own.